haplotypecaller-gvcf-gatk4.wdl

```diff
@@ -208,8 +215,15 @@ task HaplotypeCaller {
       -I ~{input_bam} \
       -L ~{interval_list} \
       -O ~{output_filename} \
-      -contamination ~{default=0 contamination} ~{true="-ERC GVCF" false="" make_gvcf} \
-      -G StandardAnnotation -G AS_StandardAnnotation -G StandardHCAnnotation
+      -contamination ~{default=0 contamination} \
+      -G StandardAnnotation -G StandardHCAnnotation ~{true="-G AS_StandardAnnotation" false="" make_gvcf} \
+      -new-qual \
+      -GQB 10 -GQB 20 -GQB 30 -GQB 40 -GQB 50 -GQB 60 -GQB 70 -GQB 80 -GQB 90 \
+      ~{true="-ERC GVCF" false="" make_gvcf} \
+      ~{bamout_arg}
+
+    # Cromwell doesn't like optional task outputs, so we have to touch this file.
```

Comment:
Really?  We definitely have this is dsde-pipelines: https://github.com/broadinstitute/dsde-pipelines/blob/develop/pipelines/dna_seq/germline/joint_genotyping/by_chromosome/JointGenotypingByChromosomePartTwo.wdl

Reply:
This note is coming from [dsde-piplines](https://github.com/broadinstitute/dsde-pipelines/blob/5cbe5dd327de0b4a29f39d704df4ee02d687e6b7/tasks/GermlineVariantDiscovery.wdl#L125) so I can't be certain. The workflow that you linked has optional output files for the workflow block but doesn't show an example of a task having an optional output. So it might be possible on a workflow level but not on a task level.